views/hooks/useParticipantsList.jsx

```diff
@@ -33,7 +33,7 @@ export default function useParticipantsList() {
             searchSubjects: normalizeSearchSubjects(formFilters.searchSubjects),
           }
         : {}),
-      ...(formFilters?.status ? { status: formFilters.status } : {}),
+      ...(formFilters?.status !== '' ? { status: formFilters.status } : {}),
```

Comment:
does this line need to change? `''` is falsey so I think the previous code should work.

Reply:
[a17906c](https://github.com/AMP-SCZ/dpdash/pull/563/commits/a17906cb0c5882bfe9973da6840eb32502b4eef6)